charts/backingservices/charts/constellation-messaging/values.yaml

```diff
@@ -13,6 +13,15 @@ imagePullPolicy: Always
 pegaMessagingPort: 3000
 pegaMessagingTargetPort: 3000
 
+# set memoryRequest &  memoryLimit to Limit memory usage for container https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-memory
```

Comment:
I would suggest taking the configuration as it is present on k8s pod spec. For example, the resources configuration in  the values.yaml can be exposed as ,
```
resources:
        requests:
          memory: "12Gi"
          cpu: 3
        limits:
          memory: "12Gi"
          cpu: 4
   ```
Sample applies to security context. This also make the templates change easy.
Please take a look at #759 
This is the pattern we have been following for the new enhancements.

Reply:
@kishorv10 updated values.yaml file as suggested.